docs/faq/known_issues.md

```diff
@@ -61,6 +61,54 @@ sudo cp /opt/ros/humble/share/pcl_ros/cmake/export_pcl_rosExport.cmake /opt/ros/
 sudo sed -i -e 's/\/opt\/ros\/humble\/include\/rclcpp;//g' /opt/ros/humble/share/pcl_ros/cmake/export_pcl_rosExport.cmake
 ```
 
+### Warnings caused by `setup.py`
+
+- Issue
+  - The following warnings happens when building CARET
+
+```sh
+/usr/local/lib/python3.10/dist-packages/setuptools/command/develop.py:40: EasyInstallDeprecationWarning: easy_install command is deprecated.
+!!
+
+        ********************************************************************************
+        Please avoid running ``setup.py`` and ``easy_install``.
+        Instead, use pypa/build, pypa/installer or other
+        standards-based tools.
+
+        See https://github.com/pypa/setuptools/issues/917 for details.
+        ********************************************************************************
+
+!!
+  easy_install.initialize_options(self)
+```
+
+```sh
+/usr/local/lib/python3.10/dist-packages/setuptools/_distutils/cmd.py:66: SetuptoolsDeprecationWarning: setup.py install is deprecated.
+!!
+
+        ********************************************************************************
+        Please avoid running ``setup.py`` directly.
+        Instead, use pypa/build, pypa/installer or other
+        standards-based tools.
+
+        See https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html for details.
+        ********************************************************************************
+
+!!
+  self.initialize_options()
+```
+
+- Cause
+  - Python3 deprecates building with `setup.py`
+  - However, `setup.py` is used to build ROS 2 by colcon.
+- Workaround
+  - This warning does not prevent CARET from working and does not require any action
```

Comment:
Add period. 
```suggestion
  - This warning does not prevent CARET from working and does not require any action.
```

Reply:
All other FAQs were written without periods. (Because they are bullet points?)

So, unlike the point made, all periods have been removed [5c4cfc9](https://github.com/tier4/CARET_doc/pull/213/commits/5c4cfc9d8ed000846418019ded201f10ed20af6c).